libs/designsystem/src/lib/components/list/list.component.scss

```diff
@@ -148,8 +142,8 @@ ion-item-options[side='end'] {
   border-bottom-width: 0;
 }
 
-:host(.has-sections) {
-  ion-list {
+:host-context(.has-sections) {
```

Comment:
When using `:host`, Angular translates it to something like `[_nghost-c0]` and if I add a h2 selector within :host, it would render something like `[_nghost-c0]   h2[_ngcontent-c0]` - which is useless when targeting child components.
The `:host-context` works more like normal CSS and I am therefore sacrificing some style encapsulation for the ability to cascade styles to child components.

Reply:
Thanks you - I learned something new today! :D